### PR TITLE
Feature/migrate to GitHub actions

### PR DESCRIPTION
**Pull request by drilonrecica**

### CODE REVIEW

File: .github/workflows/android.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        with:
+          java-version: 1.8
+      - name: Test SecureStorage
+        uses: reactivecircus/android-emulator-runner@v2
+        with:
+          api-level: ${{ matrix.api-level }}
+          target: ${{ matrix.target }}
+          arch: x86_64
+          profile: Nexus 6
+          script: ./gradlew app:connectedAndroidTest -Pandroid.testInstrumentationRunnerArguments.class=de.adorsys.android.securestoragetest.SecureStorageLogicTest --stacktrace
```
Comment (IsabellaCaspari) [inline .github/workflows/android.yml]: build a gradlew tasks

Comment (luckyhandler) [reply .github/workflows/android.yml]: ?

Comment (drilonrecica) [reply .github/workflows/android.yml]: @IsabellaCaspari This will change with the release of SecureStorage2 so I will leave it like this for the meantime

Comment (drilonrecica) [reply .github/workflows/android.yml]: Relic from travis CI. Updated it to only `./gradlew connectedAndroidTest --stacktrace`

**Pull request merged**